.travis.yml

```diff
@@ -1,20 +1,34 @@
-script:
 language: python
-python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
 
-# Travis uses Ubuntu 12.04 and 14.04. None of them have GSL2x in apt-get
-# We have to install it from source (this will make the tests slower)
-# CLASS also needs to be downloaded and compiled, which there is no easy fix for.
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install fftw3 fftw3-dev pkg-config
-#  - sudo apt-get install latexmk
-  - pip install numpy
-  - ./install-gsl.sh
-  - python class_install.py
+# Caching all the depencencies so that they don't have to be redownloaded and
+# compiled everytime
+cache:
+  directories:
+    - $HOME/build/EiffL/CCL/build/GSL/src
```

Comment:
Travis has a `$TRAVIS_BUILD_DIR` environment variable, which points to the default Travis working directory -- it might be a bit cleaner to put the cached libraries in there instead.

Reply:
sure, thanks for the pointer, will update the Dockerfile